src/java/picard/analysis/CollectQualityYieldMetrics.java

```diff
@@ -45,14 +45,41 @@
  *
  * @author Martha Borkan
  */
+
+
 @CommandLineProgramProperties(
-        usage = "Collects quality yield metrics, a set of metrics that quantify the quality and yield of sequence data from a " +
-                "SAM/BAM input file.",
-        usageShort = "Collects a set of metrics that quantify the quality and yield of sequence data from the provided SAM/BAM",
+        usage = CollectQualityYieldMetrics.USAGE_SUMMARY + CollectQualityYieldMetrics.USAGE_DETAILS,
+        usageShort = CollectQualityYieldMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectQualityYieldMetrics extends CommandLineProgram {
-
+    static final String USAGE_SUMMARY = "Collects a set of metrics that indicate the quantity of reads that pass quality thresholds and Illumina-specific filters.  ";
+    static final String USAGE_DETAILS = "This tool helps the user determine the overall quality of a set of reads " +
+            "determining how many reads of a run have a minimum base-quality score and pass filters.  Either the original (OQ) or the recalibrated base quality " +
+            "scores can be used.  The recalibrated base quality scores are obtained from the Base Quality Score Recalibration " +
```

Comment:
Now reads: 
"This tool enables the user determine the overall quality of reads within a read group bam file, where the read group is a subset of a lane of sequence data.  The output indicates the total numbers of bases within a read group that have a minimum base-quality score and pass quality filters (see below)."